Makefile

```diff
@@ -52,6 +50,14 @@ tpu-tgi:
 	             -t huggingface/optimum-tpu:$(VERSION)-tgi .
 	docker tag huggingface/optimum-tpu:$(VERSION)-tgi huggingface/optimum-tpu:latest
 
+tpu-tgi-old:
```

Comment:
why "old"?

Reply:
I forgot to remove it, thanks for spotting it